common/buildcraft/lib/misc/WorkerThreadUtil.java

```diff
@@ -66,29 +66,29 @@
      * gone wrong, and will notify the log that a task took too long. If it goes on for longer than 10s then it will
      * make a big error in the log. */
     public static void executeWorkTask(Runnable task) {
-        executeWorkTask(new CallableDelegate(task));
+        executeWorkTask(new CallableDelegate(task), true);
     }
 
     /** Executes a task. If this is in debug mode then If this takes longer than 30ms the it assumes that something has
      * gone wrong, and will notify the log that a task took too long. If it goes on for longer than 10s then it will
      * make a big error in the log. */
-    public static <T> Future<T> executeWorkTask(Callable<T> task) {
+    public static <T> Future<T> executeWorkTask(Callable<T> task, boolean shouldMonitor) {
```

Comment:
alright just thought this would be the least invasive option to not have my task killed